pydatastructs/utils/misc_util.py

```diff
@@ -148,6 +148,9 @@ def add_children(self, *children):
         for child in children:
             self.children.append(child)
 
+    def __str__(self):
+        return str((self.key, self.data))
+
```

Comment:
```suggestion
```
Already present in parent class, `TreeNode`.

Reply:
gives error because there is no left or right node in MAryTreeNode